sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/ServiceAPITest.groovy

```diff
@@ -197,4 +209,290 @@ class ServiceAPITest extends APISpec {
         response.getHeaders().getValue("x-ms-version") == "2019-02-02"
     }
 
+    def "Restore file system"() {
+        given:
+        def cc1 = primaryDataLakeServiceClient.getFileSystemClient(generateFileSystemName())
+        cc1.create()
+        def blobName = generatePathName()
+        cc1.getFileClient(blobName).upload(defaultInputStream.get(), 7)
+        cc1.delete()
```

Comment:
maybe add an exists check to ensure it actually was deleted?

Reply:
Done